dev-vcs/mercurial/mercurial-5.3.1.ebuild

```diff
@@ -0,0 +1,171 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6  python3_7 )
```

Comment:
You've got double space there. What about 3.8?

Reply:
https://github.com/gentoo/gentoo/pull/14943#issuecomment-604960742